### PR TITLE
Feature/cint aniseed v1 source

### PR DESCRIPTION
**Pull request by avullo**
This covers work for JGI provided Ciona intestinalis data

### CODE REVIEW

File: lib/Bio/EnsEMBL/Versioning/Pipeline/Downloader/JGI.pm [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+package Bio::EnsEMBL::Versioning::Pipeline::Downloader::JGI;
+
+use Moose;
+
+extends 'Bio::EnsEMBL::Versioning::Pipeline::Downloader';
+
+has uri => (
+  isa => 'Str', 
+  is => 'ro',
+  default => 'ftp://ftp.jgi-psf.org/pub/JGI_data/Ciona/v1.0/',
```
Comment (nerdstrike) [inline lib/Bio/EnsEMBL/Versioning/Pipeline/Downloader/JGI.pm]: I think we might need to consider the other ciona versions. e! website says assembly KH from 2011, whereas v1.0 is from 2003 or so. I'll ask around and let you know.

Comment (avullo) [inline lib/Bio/EnsEMBL/Versioning/Pipeline/Downloader/JGI.pm]: Sounds sensible, anyway the current xref pipeline is not considering them

**Comment by magaliruffier** [comment]
The Tunicates community does not want to use the more recent assembly, because they reckon it is of poor quality and any annotation produced on it is bad
Hence we link to the annotation produced on JGI 1.0
I have updated the list of xref sources to record this

**Comment by avullo** [comment]
That's great, another source in the bag :+1:

**Pull request merged**